app/views/shared/_profile_card_content.html.erb

```diff
@@ -29,25 +29,17 @@
           </div>
         </li>
       <% end %>
-      <% if actor.employment_title.present? %>
+      <% if actor.profile.respond_to?(:work) && actor.profile.work.present? %>
```

Comment:
@benhalpern When we decide to consolidate the 3 employment profile fields into the single `work` field you opted for a profile field, not a column on the `profiles` table. This necessitates this special case, which will stop working if an admin e.g. deletes the original field and later recreates it with another label like "Employment" or "Occupation". 

On the profile page this is not a problem, as fields with a `display_area` of `header` get special treatment, independent of their name. This her feels a bit iffy though, so I wonder if we actually should change this to something similar to what we have in `app/views/users/_metadata.html.erb`, i.e. all header fields become part of the preview card.

```ruby
<% if (header_fields = @profile.ui_attributes_for(area: :header)).present? %>
  <div class="profile-header__bottom fs-base">
    <% header_fields.sort.each do |title, value| %>
      <% next if Profile.special_attributes.include?(title) %>
      <div class="crayons-definition">
        <strong class="crayons-definition__title">
          <%= sanitized_sidebar title.titleize %>
        </strong>
        <p class="crayons-definition__value">
          <%= sanitized_sidebar value %>
        </p>
      </div>
    <% end %>
  </div>
<% end %>
```

Reply:
Yes, I agree with this approach.

My only hesitation here is... We don't currently limit the _total number_ of `display_area=header` fields at the moment, right?

IMO a hard limit of **three** total header fields via custom validation would make sense to solve for a number of potential problems on different views if we can't assume a certain limit.

Reply:
Yes, that was one of the concerns I had too, we can go with a custom validation to limit this to 3. Thanks!